middleware/authenticate.js

```diff
@@ -1,28 +1,32 @@
-const jwt = require('jsonwebtoken');
-const {User} = require("../services/authService");
-
-const isAuthenticated = async (req, res, next) => {
-    try {
-        const authHeader = req.headers.authorization;
-        const token = authHeader && authHeader.split(' ')[1]
-        const Token = jwt.verify(token, process.env.ACCESS_TOKEN_SECRET, {
-            expiresIn: process.env.JWT_EXPIRE,
-            algorithm: 'HS256'
-        });
-        if (token === null) res.status(401)
-
-        const user = await User.retrieve(Token.userEmail);
-        req.user = user;
-
-        console.log()
-        res.header({authorization: authHeader});
-
-        next();
-    } catch (err) {
-        res.status(403).json({
-            message: err.message
-        })
-    }
-}
-
-module.exports = {isAuthenticated}
\ No newline at end of file
+// const jwt = require('jsonwebtoken');
+// const {User} = require("../services/authService");
+// const express = require('express')
+// const {header} = require("express-validator");
+// const authJwt = new express.Router();
+//
+// // authJwt.post('/refreshToken', async (req, res) => {
+// //     const refreshToken = req.body.refreshToken;
+// //     if (refreshToken == null) return res.sendStatus(401)
+// //     // const user = await User.retrieve({refreshToken: refreshToken})
+// //     // if (!user) return res.sendStatus(403)
+// //     jwt.verify(refreshToken, process.env.REFRESH_TOKEN_SECRET, (err, user) => {
+// //         if (err) return res.sendStatus(403)
+// //         const accessToken = generateAccessToken({email: user.email})
+// //         //  res.setHeader("authorization", "Bearer" + " " + accessToken)
+// //         req.token = {accessToken: accessToken};
```

Comment:
do not commit commented-out code.

BTW I don't see any review of the other teammates.